kmk/handlers/sequences.py

```diff
@@ -42,7 +42,7 @@ def send_string(message):
     seq = []
 
     for char in message:
-        kc = KC[char]
+        kc = KC.__getattr__(char.upper())
```

Comment:
Somewhat of a lazy fix, I made an attempt to get `KeyAttrDict` to also create missing key entries when accessed via `__getitem__` but I couldn't figure it out so this will have to do unless someone else has a better idea.

Seems to work fine on my end testing on a Feather RP2040

Reply:
Preference to using `getattr(KC, char.upper())` here instead of directly calling the dunder (which is a bit of an implementation detail) but otherwise this looks good :+1: 

Thanks for doing this!